msalautomationapp/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
-def msalVersion = "5.0.0"
+def msalVersion = "5.+"
```

Comment:
lets make it specific version to this release.

Reply:
Talked with Amit offline; keeping these to +.